packages/client/components/PokerEstimateHeaderCardJira.tsx

```diff
@@ -83,7 +84,7 @@ const PokerEstimateHeaderCardJira = (props: Props) => {
   const {serviceTaskId, story} = stage
   const [isExpanded, setIsExpanded] = useState(false)
   const descriptionRef = useRef<HTMLDivElement>(null)
-  const maxHeight = descriptionRef.current?.scrollHeight ?? 1000
+  const maxHeight = Math.min(descriptionRef.current?.scrollHeight ?? 300, 300)
```

Comment:
I’ll need to understand this, but I wonder if we find a way that both the card and the voting area could be scrolled if things got really tight; I’m looking at this locally and need to come back with an idea